packages/web-components/public/switches.ts

```diff
@@ -0,0 +1,25 @@
+import { fillColor, StandardLuminance, SwatchRGB } from '../src/index';
+
+export function toggleBgMode() {
+  const bgChecked = document.getElementById('luminance-switch')!.classList.contains('checked');
+  if (bgChecked) {
+    fillColor.setValueFor(
+      document.body,
+      SwatchRGB.create(StandardLuminance.LightMode, StandardLuminance.LightMode, StandardLuminance.LightMode),
+    );
+  } else {
+    fillColor.setValueFor(
+      document.body,
+      SwatchRGB.create(StandardLuminance.DarkMode, StandardLuminance.DarkMode, StandardLuminance.DarkMode),
+    );
+  }
+}
+
+export function toggleLtr() {
+  const ltrChecked = document.getElementById('ltr-switch')!.classList.contains('checked');
+  if (ltrChecked) {
+    document.querySelector<HTMLElement>('.docs-story')!.setAttribute('style', 'direction:ltr;');
```

Comment:
We need to set the `direction` design token as well for components like horizontal scroll, switch, and tree view. There should probably be a way to make sure these stay in sync.